### PR TITLE
Update bundle_de.properties to latest bundle

### PR DESCRIPTION
**Pull request by hype11**
sort key/value entries like bundle.properties (master) 
add new keys from bundle.properties

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_de.properties [hunk 1]
```diff
@@ -25,136 +29,175 @@ command.org.eclipse.ui.navigate.collapseAll.name                       = Alle zu
...
-keyword.org.jkiss.dbeaver.core.keyword.pref.editor.content.label = lob blob clob
-keyword.org.jkiss.dbeaver.core.keyword.pref.editor.entity.label  = object entity schema catalog table package column
-keyword.org.jkiss.dbeaver.core.keyword.pref.editor.erd.label     = erd diagram
-keyword.org.jkiss.dbeaver.core.keyword.pref.editor.sql.label     = sql script commit fetch timeout
-keyword.org.jkiss.dbeaver.core.keyword.pref.ui.general.label     = update http proxy
 
-maven.respository.central          = Zentrales Repository
-maven.respository.central.unsecure = Zentrales Repository (unsicher)
+menu.navigate     = Navigieren
+menu.database     = &Database
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_de.properties]:
```suggestion
menu.database     = &Datenbank
```

Comment (hype11) [reply plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_de.properties]: I thought &Database is a placeholder which is replaced by database name at runtime. Is it a normal translation?

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_de.properties]: yes

**Review by LonwoLonwo — APPROVED** [review]

**Review by E1izabeth — APPROVED** [review]

**Comment by Matvey16** [comment]
Verified

**Pull request merged**